execution_engine/CHANGELOG.md

```diff
@@ -27,6 +27,13 @@ All notable changes to this project will be documented in this file.  The format
 
 
 
+## 2.0.1
+
+### Security
+* Implement checks before preprocessing Wasm to avoid potential OOM when initializing table section.
+* Implement checks before preprocessing Wasm to avoid references to undeclared functions or globals.
```

Comment:
Didn't we have something with gas counter as well?

Reply:
Forgot about adding this one entry in 1.4.8. Addressed in 77b55032f5c5e32a37a0c119548eb48ac1dc332c